test/js/test.js

```diff
@@ -233,6 +235,9 @@ function makeAccountKeyPair(answers) {
 }
 
 function register(answers) {
+  if (state.abortStep === "register") {
```

Comment:
In the spirit of "You Ain't Gonna Need It", let's leave out all of these abortStep if statements from the Node test except the one we actually use - the `startChallenge` one.